waku/node/v2/message_store.nim

```diff
@@ -0,0 +1,222 @@
+import 
+  os, 
+  sqlite3_abi,
+  waku_types,
+  chronos, chronicles, metrics, stew/results,
+  libp2p/crypto/crypto,
+  libp2p/protocols/protocol,
+  libp2p/protobuf/minprotobuf,
+  libp2p/stream/connection,
+  stew/results, metrics
+
+{.push raises: [Defect].}
+
+# The code in this file is an adaptation of the Sqlite KV Store found in nim-eth.
+# https://github.com/status-im/nim-eth/blob/master/eth/db/kvstore_sqlite3.nim
+#
+# Most of it is a direct copy, the only unique functions being `get` and `put`.
```

Comment:
It'd be nice if we could keep this the custom stuff in a separate file, but maybe this is tricky? Not sure

Reply:
I don't think it makes sense because its all part of the db. might be tricky.

Reply:
at some point, we might make a "nice" nim api for sqlite out of these various usages and then refactor - wouldn't bother until then, really

Reply:
aight let's leave it like this for now